preview-maker.js

```diff
@@ -79,33 +101,127 @@ const makePrev = async (themeBuffer, themeName, themeAuthor, template) => {
+    }
+    for (const PreviewBack of getElementsByClassName(preview, "PreviewBack")) {
+        fillHSL(PreviewBack, `hsl(${accentHue}, 100%, 90%)`);
+    }
+    for (const ChatShadow of getElementsByClassName(preview, "ChatShadow")) {
+        fillHSL(ChatShadow, `hsl(${accentHue}, 100%, 2%)`);
+    }
+    if (theme[`chat_outBubbleGradient`] != undefined) {
+        for (const PreviewBack of getElementsByClassName(preview, "PreviewBack")){
+            fill(PreviewBack, {red:0,green:0,blue:0,alpha:0});
+        }
+    }
+    for (const avatar of avatars){
+        const windowBackgroundWhite = theme['windowBackgroundWhite'] || defaultVariablesValues['windowBackgroundWhite']
+        if (Math.abs(theme[avatar].red - windowBackgroundWhite.red) + Math.abs(theme[avatar].green - windowBackgroundWhite.green) + Math.abs(theme[avatar].blue - windowBackgroundWhite.blue) < 15){
+            for (const ava of getElementsByClassName(preview, avatar)){
+                fill(ava, theme['avatar_text'] || defaultVariablesValues['avatar_text'])};
+                for (const avashadow of getElementsByClassName(preview, `${avatar}Shadow`)) {
+                    const choose = theme['avatar_text'] || defaultVariablesValues['avatar_text'];
+                    const hslChoose = rgbToHsl(choose)
+                    fillHSL(avashadow,`hsl(${hslChoose.hue}, ${hslChoose.saturation}%, ${hslChoose.lightness - hslChoose.lightness > 25 && rgbToHsl(windowBackgroundWhite).lightness < 10 ? -20 : 40 }%)`)
+                }
+            } else {
+                    for (const avashadow of getElementsByClassName(preview, `${avatar}Shadow`)) {
+                        const choose = theme[avatar] || defaultVariablesValues[avatar];
+                        const hslChoose = rgbToHsl(choose)
+                        fillHSL(avashadow,`hsl(${hslChoose.hue}, ${hslChoose.saturation}%, ${hslChoose.lightness - hslChoose.lightness > 25 && rgbToHsl(windowBackgroundWhite).lightness < 10 ? -20 : 40 }%)`);
+                    }
+                }
+            }
+    const chat_inLoader = theme['chat_inLoader'] || defaultVariablesValues['chat_inLoader'];
+    const chat_inBubble = theme['chat_inBubble'] || defaultVariablesValues['chat_inBubble'];
+    const chat_outLoader = theme['chat_outLoader'] || defaultVariablesValues['chat_outLoader'];
+    const chat_outBubble = theme['chat_outBubble'] || defaultVariablesValues['chat_outBubble'];
+    if (Math.abs(chat_inLoader.red - chat_inBubble.red) +
+        Math.abs(chat_inLoader.green - chat_inBubble.green) +
+        Math.abs(chat_inLoader.blue - chat_inBubble.blue) <
+        15) {
+        for (const chat_inLoader of getElementsByClassName(preview, "chat_inLoader")) {
+            fill(chat_inLoader, theme["chat_inMediaIcon"] || defaultVariablesValues["chat_inMediaIcon"]);
+        }
+    }
+    if (Math.abs(chat_outLoader.red - chat_outBubble.red) +
```

Comment:
please put blank lines between semantical blocks of code. it's easier to read code that way